src/project/mesa3d_desktop_intel.rs

```diff
@@ -4,14 +4,16 @@
 use super::*;
 
 #[derive(Default)]
-pub struct MesaDesktopIntel();
+pub struct Mesa3DDesktopIntel {
+    src_path: PathBuf,
+}
 
-impl Project for MesaDesktopIntel {
+impl Project for Mesa3DDesktopIntel {
     fn get_name(&self) -> &'static str {
-        "mesa_desktop_intel"
+        "mesa3d/desktop-intel"
```

Comment:
If you want that name, the CI needs to be updated to support it. Right now it does not support sub-folder I believe.

Reply:
I have attempted to fix this issue with the commit `.github: Allow scripts to exist in a subdirectory`. We will see if it works :)